bin/desi_proc_joint_fit

```diff
@@ -370,9 +371,21 @@ if args.obstype in ['SCIENCE']:
             saved_stdfile = findfile('stdstars', args.night, args.expids[0], spectrograph=sp)
             for expid in args.expids[1:]:
                 new_stdfile = findfile('stdstars', args.night, expid, spectrograph=sp)
-                cmd = f'ln -s {saved_stdfile} {new_stdfile}'
-                log.info(f'Sym Linking jointly fitted stdstar file: {saved_stdfile}  to  {new_stdfile}')
+                new_dirname, new_fname = os.path.split(new_stdfile)
+                log.debug("Path exists: {}, file exists: {}, link exists: {}".format(os.path.exists(new_stdfile),
+                                                                                     os.path.isfile(new_stdfile),
+                                                                                     os.path.islink(new_stdfile)))
+                relpath_saved_std = os.path.relpath(saved_stdfile, new_dirname)
+                log.debug(f'Sym Linking jointly fitted stdstar file: {new_stdfile} '+\
+                         f'to existing file at rel. path {relpath_saved_std}')
+                cmd = f'ln -s {relpath_saved_std} {new_stdfile}'
+                cmdlist = cmd.split(' ')
+                log.info(str(cmdlist))
```

Comment:
What is the advantage to splitting the command and then converting *back* into a string in order to log it?

Reply:
Also `cmd.split(' ')` is *not* the same as splitting on space characters:
```python
>>> foo = 'a  b'
>>> foo.split()
['a', 'b']
>>> foo.split(' ')
['a', '', 'b']
```
If you accidentally introduce an extra space character, `.split(' ')` does not behave as expected.

If you have to construct a command as a string `shutil.split()` is probably the safest way to go.  I strongly recommend constructing the command explicitly as a string.  You have absolutely nothing to lose here:
```python
cmdlist = ['ln', '-s', relpath_saved_std, new_stdfile]
```
Or you could use `os.symlink`.  I really, really cannot imagine why you wouldn't.

Reply:
It's a good question. str(cmdlist) gives a str form of the list being given to the following function. I wanted to ensure that the split worked appropriately and verify that we're giving the runcmd function the proper thing it is expecting. It is not the same as rejoining the list back into the original.

Reply:
Sorry:

>  I strongly recommend constructing the command explicitly as a string.

I meant "as a `list()`".

Reply:
These are removed using the new implementation. Thanks for the comments